NebulaModel/MultiplayerOptions.cs

```diff
@@ -7,6 +7,21 @@ namespace NebulaModel
     [System.Serializable]
     public class MultiplayerOptions : ICloneable
     {
+        [DisplayName("Nickname")]
+        public string Nickname { get; set; } = "Player";
```

Comment:
We could use `AccountData.me.userName` as default value

Reply:
I wanted to do this but the config generates before AccountData.me.userName is populated

Reply:
Do you know when that value gets populated and then auto-set this value to it? Or is that not viable (performance, or the population is too late)?

Reply:
I could keep the default null and make a postfix patch for PARTNER.Login() and do Config.Options.Nickname = AccountData.me.userName ?? "Player"; there I suppose.

Would this be desireable?

Some games, though usually older ones, with official multiplayer support default to just calling the player "Player" so I figured this was just the cleanest approach but it's up to you folks.